### ISSUE TITLE
Android not working with Expo 52

### ISSUE THREAD

**Issue by deelo55** [shakebugs/shake-react-native#8]
When running with android and expo 52 (works fine with expo 51) I get the following error:

`Error: Exception in HostObject::get for prop 'RNShake': com.facebook.react.internal.turbomodule.core.TurboModuleInteropUtils$ParsingException: Unable to parse @ReactMethod annotations from native module: RNShake. Details: Module exports two methods to JavaScript with the same name: "show`

```
import Shake from "@shakebugs/react-native-shake";
import { StatusBar } from "expo-status-bar";
import { useEffect } from "react";
import { Platform, StyleSheet, Text, View } from "react-native";
export default function App() {
  useEffect(() => {
    const initShake = async () => {
      if (Platform.OS === "ios") {
        Shake.start("xxx");
      }
      if (Platform.OS === "android") {
        Shake.start("xxx");
      }
    };
     initShake();
  }, []);

  return (
    <View style={styles.container}>
      <Text>Open up App.tsx to start working on your app!</Text>
      <StatusBar style="auto" />
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: "#fff",
    alignItems: "center",
    justifyContent: "center",
  },
});

```

**Comment by mstanic-shake** [comment]
In Expo 52, New architecture is enabled by default. Shake still doesn't support new architecture, we are looking forward to support it in next 2-3 weeks.

**Comment by gezquinndesign** [comment]
@mstanic-shake Does this mean that Shake now works on Expo? I must have missed the announcement.

**Comment by mstanic-shake** [comment]
@gezquinndesign We don't support it officially, but probably you can add it on your own somehow. Although i guess we'll add official support for Expo in near future.

For the OG, we have migrated now to new architecture but still in qa. So you can expect new version soon.

**Comment by mstanic-shake** [comment]
We have released Shake RN 17.1.0 version which supports only new architecture. For old architecture use older version 17.0.1

**Comment by steflsd** [comment]
Hi. We're using expo52 with 'newArchEnabled: false,' as we have some other dependencies that we cant upgrade.  After upgrading to 17.1.0 the android build fails (IOS works!) 

```
e: file:///home/expo/workingdir/build/node_modules/@shakebugs/react-native-shake/android/src/main/java/com/shakebugs/react/ShakePackage.kt:12:42 Type mismatch: inferred type is ShakeModule but NativeModule? was expected
```

Downgrading back to 17.0.1 also doesnt seem to work either now (maybe some other dependency change, not sure).
